lib/ui/compositing.dart

```diff
@@ -108,7 +108,7 @@ abstract class _EngineLayerWrapper implements EngineLayer {
   bool _debugCheckNotUsedAsOldLayer() {
     assert(
         !_debugWasUsedAsOldLayer,
-        'Layer $runtimeType was previously used as oldLayer.\n'
+        'Layer $runtimeType#${hashCode.toUnsigned(20).toRadixString(16).padLeft(5, '0')} was previously used as oldLayer.\n'
```

Comment:
leave a comment here saying this should match shortHash in the framework?

Reply:
done